DominoLibrary/Board.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace DominoLibrary;
+﻿using Utils;
```

Comment:
I've never used this class, I didn't even know it existed 😅 Can you explain to me what it is and what made you want to use it? 🤓🙏

Reply:
It's a namespace created by me. It's the namespace which contains CircularList & Node classes

Reply:
Ok!